### PR TITLE
EMP-644: Refactoring TaskImageFilesModel to CrmFormDetailsModel and update data source

### PR DESCRIPTION
**Pull request by tanovellino**
EMP-644: Refactoring TaskImageFilesModel to CrmFormDetailsModel and update data source

## What

[Link to story](https://dsdmoj.atlassian.net/browse/EMP-644)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [x] Tests should be passing: `./gradlew test`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [x] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [x] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm14ControllerTest.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ class Crm14ControllerTest {
     private SoftAssertions softly;
 
     @Autowired
-    TaskImageFilesRepository taskImageFilesRepository;
+    CrmFormDetailsRepository detailsRepository;
```
Comment (gpjustice) [inline equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm14ControllerTest.java:48]: ` crmFormDetailsRepository` - As we are factoring, just trying to put the correct name.

File: equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm14ControllerTest.java [hunk 2]
```diff
@@ -75,11 +75,11 @@ void preTest() {
                 FileInputStream fis = new FileInputStream(testFile);
                 JSONObject mockedCrmFileJson = new JSONObject(IOUtils.toString(fis, StandardCharsets.UTF_8));
                 byte[] fileDataByte = XML.toString(mockedCrmFileJson).getBytes(StandardCharsets.UTF_8);
-                TaskImageFilesModel taskModel = new TaskImageFilesModel();
+                CrmFormDetailsModel taskModel = new CrmFormDetailsModel();
```
Comment (gpjustice) [inline equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm14ControllerTest.java:78]: same as above - `crmFormDetailsModel`

File: equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm4ControllerTest.java [hunk 3]
```diff
@@ -43,7 +43,7 @@ class Crm4ControllerTest {
     private SoftAssertions softly;
 
     @Autowired
-    TaskImageFilesRepository taskImageFilesRepository;
+    CrmFormDetailsRepository detailsRepository;
```
Comment (gpjustice) [inline equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm4ControllerTest.java:46]: same as bove `crmFormDetailsRepository`

File: equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm4ControllerTest.java [hunk 4]
```diff
@@ -63,11 +63,11 @@ void preTest() throws IOException {
                 FileInputStream fis = new FileInputStream(testFile);
                 JSONObject mockedCrmFileJson = new JSONObject(IOUtils.toString(fis, StandardCharsets.UTF_8));
                 byte[] fileDataByte = XML.toString(mockedCrmFileJson).getBytes(StandardCharsets.UTF_8);
-                TaskImageFilesModel taskModel = new TaskImageFilesModel();
+                CrmFormDetailsModel taskModel = new CrmFormDetailsModel();
```
Comment (gpjustice) [inline equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm4ControllerTest.java:66]: same as bove `crmFormDetailsModel`

File: equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm14ControllerTest.java (hunk 2 above)
Comment (tanovellino) [reply equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm14ControllerTest.java:78]: noted.. changing

File: equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm4ControllerTest.java (hunk 3 above)
Comment (tanovellino) [reply equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm4ControllerTest.java]: changed

**Pull request merged**